osm_fieldwork/json2osm.py

```diff
@@ -441,15 +437,21 @@ def main():
 
     # if verbose, dump to the terminal.
     if args.verbose is not None:
-        log_level = logging.DEBUG
-        log_stream = sys.stdout
-
-    logging.basicConfig(
-        level=log_level,
-        format=("%(threadName)10s - %(name)s - %(levelname)s - %(message)s"),
-        datefmt="%y-%m-%d %H:%M:%S",
-        stream=log_stream,
-    )
+        logging.basicConfig(
```

Comment:
Hey @spwoodcock , I have made the changes now.

The log_level and log_stream variables are now replaced with hardcoded basicConfig versions.

Because the urllib configuration depends on log_level, I moved it to the args.verbose block. 

I left the ODKCentral.py code because it doesnt have a verbose argument and instead depends on an environment variable.